generators/ruby-v2/base/src/project/RubyProject.ts

```diff
@@ -443,7 +443,7 @@ class VersionFile {
 
     public toString(): string {
         const seedName = this.context.getRootModuleName();
-        const version = this.context.getVersionFromConfig();
+        const version = this.context.getVersionFromConfig(this.context.logger);
```

Comment:
it's weird to me that we're calling this.context.<function> and needing to pass in this.context.<field>, can it not just get the logger from the context in the function?